class Main
{
	public static void main(String[] args) {
	    try{
	        int dividebyzero=5/0;
	        System.out.println("Rest of code in try block");
	    }
	    catch(ArithmeticException e){
	        System.out.println("ArithmaticExeption="+e.getMessage());
	    }
		//System.out.println("Hello WorldðŸ˜Š");
	}
}
